pkgs/applications/audio/in-formant/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, cmake, lib, fetchFromGitHub, qt5, fftw, libtorch-bin, portaudio, eigen
+, xorg, pkg-config, autoPatchelfHook, soxr
+}:
+
+stdenv.mkDerivation rec {
+  pname = "in-formant";
+  version = "2021-06-30";
```

Comment:
This should be prefixed with unstable-

Reply:
Just a note for next time since it's already merged?